The definition of TCL_REG_BOSONLY has moved from tcl.h to tclInt.h in
Tcl 8.5a6 and later. This patch makes sure that the right header file
is included. For earlier versions of Tcl tcl.h is included from
tclInt.h anyway.
-- 
Jonas Karlsson

--- expect-5.44.1/exp_inter.c	2005-01-22 00:17:12.000000000 +0100
+++ expect-5.44.1/exp_inter.c.new	2007-10-25 22:27:39.000000000 +0200
@@ -35,7 +35,7 @@
 
 #include <ctype.h>
 
-#include "tcl.h"
+#include "tclInt.h"
 #include "string.h"
 
 #include "exp_tty_in.h"
